src/modality-specific-files/electromyography.md

```diff
@@ -0,0 +1,652 @@
+# Electromyography
+
+Support for Electromyography (EMG) was developed as a
+[BIDS Extension Proposal](../extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../introduction.md#citing-bids) on how to appropriately credit
+this extension when referring to it in the context of the academic literature.
+
+!!! example "Example datasets"
+
+    Electromyography datasets formatted according to this specification are available on
+    the [BIDS examples repository](https://github.com/bids-standard/bids-examples#emg)
+    and can be emulated when curating new datasets.
+
+## EMG data
+
+{{ MACROS___make_filename_template(
+"raw",
+datatypes=["emg"],
+suffixes=["emg", "events"])
+}}
+
+EMG device manufacturers use a variety of formats for storing raw data, and there is
+no single standard that all researchers agree on. For BIDS, EMG data MUST be
+stored in one of the following formats:
+
+| **Format**                                                         | **Extension(s)**         | **Description**                                                                                                                                                                                      |
+| ------------------------------------------------------------------ | ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Biosemi data format](https://www.biosemi.com/faq/file_format.htm) | `.bdf`                   | Each recording consists of a single `.bdf` file. [`bdf+`](https://www.teuniz.net/edfbrowser/bdfplus%20format%20description.html) files are permitted. The capital `.BDF` extension MUST NOT be used. |
+| [European data format](https://www.edfplus.info/)                  | `.edf`                   | Each recording consists of a single `.edf` file. [`edf+`](https://www.edfplus.info/specs/edfplus.html) files are permitted. The capital `.EDF` extension MUST NOT be used.                           |
+
+EDF, EDF+, BDF, and BDF+ are all open data formats with broad support  in various programming languages for reading and writing the files. BDF and BDF+ formats store data samples using 3 bytes instead of 2 bytes as in EDF and EDF+ formats, allowing for greater resolution. EDF+/BDF+ accommodate more header metadata than EDF/BDF, and support storing event or annotation information in the file. Thus it is RECOMMENDED to use the BDF+ data format.
```

Comment:
```suggestion
EDF, EDF+, BDF, and BDF+ are all open data formats with broad support in various programming languages for reading and writing the files. BDF and BDF+ formats store data samples using 3 bytes instead of 2 bytes as in EDF and EDF+ formats, allowing for greater resolution. EDF+/BDF+ accommodate more header metadata than EDF/BDF, and support storing event or annotation information in the file. Thus it is RECOMMENDED to use the BDF+ data format.
```

Reply:
what is the BDF+ format? I was not able to find any documentation for it.

Reply:
> what is the BDF+ format? I was not able to find any documentation for it.

The link is in the table that immediately precedes this paragraph. https://www.teuniz.net/edfbrowser/bdfplus%20format%20description.html